frontend/src/model/beta/overview/FormEditPage.tsx

```diff
@@ -34,12 +35,15 @@ export default function FormEditPage({ model }: FormEditPageProps) {
 
   async function onSubmit() {
     if (schema) {
+      setErrorMessage('')
       setLoading(true)
       const data = getStepsData(splitSchema, true)
       const res = await putModelCard(model.id, data)
       if (res.status && res.status < 400) {
```

Comment:
Can we use `if (!res.ok)`

Reply:
See https://github.com/gchq/Bailo/pull/950#discussion_r1425246559